samples/Mvc.RestrictiveFileUpload/web.Debug.config

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
delete debug.config and release.config. The tooling only generated them by mistake.